website/firebase.json

```diff
@@ -312,11 +312,6 @@
         "destination": "https://firebase.google.com/docs/storage/flutter/start",
         "type": 301
       },
-      {
-        "source": "/docs/firestore-odm/*",
-        "destination": "https://github.com/firebase/flutterfire/tree/master/packages/cloud_firestore_odm",
-        "type": 301
-      },
```

Comment:
I think we should keep documentation just in case? Not sure if this file is still used?

Reply:
Moved here I guess: https://github.com/FirebaseExtended/firestoreodm-flutter/tree/main/docs